schemas/_common-schema.json

```diff
@@ -180,6 +180,19 @@
       },
       "readOnly": true
     },
+    "ci_visibility": {
```

Comment:
I wonder how this section (property) would be used from let's say mobile sdk perspective ? Is this meant to be used on all platforms ?

Reply:
I hadn't thought about it, but yeah, I don't see why not, there are no technology leakage. We currently use the trace id for the cypress poc, which could be reused for others